lib/features/manage_account/presentation/extensions/vacation_response_extension.dart

```diff
@@ -78,15 +77,14 @@ extension VacationResponseExtension on VacationResponse {
     UTCDate? fromDate,
     UTCDate? toDate,
     String? subject,
-    String? textBody,
     String? htmlBody
   }) {
     return VacationResponse(
       isEnabled: isEnabled ?? this.isEnabled,
       fromDate: fromDate ?? this.fromDate,
       toDate: toDate ?? this.toDate,
       subject: subject ?? this.subject,
-      textBody: textBody ?? this.textBody,
+      textBody: null,
```

Comment:
Don't need set value for it if it is null

Reply:
Done